### PR TITLE
updated pull_request_template.md

### PR DESCRIPTION
**Pull request by GundaSudarrshan**
added .github/pull_request_template.md
and added docs/pull_request_templates.md files in the main folders

### CODE REVIEW

**Comment by Vikash-8090-Yadav** [comment]
I guess u haven't pull the  repo to the newest one as there is lot's of changes made in the readme.md 
type git pull in the terminal to fetch the chnages also .

**Review by Vikash-8090-Yadav — DISMISSED** [review]
please correct line no 24 and 25
open the bracket's
and please add the screen shot section also at the last . 
Thank u :smile:

**Comment by Vikash-8090-Yadav** [comment]
Also your P.R is 25 commit behind please update that  one also

**Comment by GundaSudarrshan** [comment]
> Also your P.R is 25 commit behind please update that one also before submitting the P.R

for me it shows upto date when using gitpull

**Comment by Vikash-8090-Yadav** [comment]
Your branch is 1 commit behind fetch the changes

**Comment by GundaSudarrshan** [comment]
@Vikash-8090-Yadav  yes I have updated

File: .github/pull_request.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+# Type of Change 📝
+please delete options that are not relevant. In order to tick the checkbox just put x inside them for example [x] like this
+
+- [ ] Add a new project
+- [ ] Modified or enchanced the existing projects
+- [ ] Improve the readme of any project
+
+# Domain of Contribution 📊
+please delete options that are not relevant. In order to tick the check box just put x inside them for example [x] like this
```
Comment (Vikash-8090-Yadav) [inline .github/pull_request.md:15]: Remove the 16 , 17 and 18 line and add  this 

[ ] Project using only HTML  ,CSS , JAVASCRIPT , WEB3.js
[ ] Project using only HTML , CSS , JAVASCRIPT , ETHER.js
[ ] Project using only HTML  , CSS , PYTHON , WEB3.py
[ ] Project using only GANACHE , TRUFFLE , HARDHAT , REMIX 
[ ] if Project use another stack and Tool feel free to mention

**Comment by Vikash-8090-Yadav** [comment]
Please add a screenshot part below of the PR  template a/c to the pic shared in the issue section #17

**Comment by Vikash-8090-Yadav** [comment]
after this all set to merge :smile:

**Comment by Vikash-8090-Yadav** [comment]
@GundaSudarrshan  please do this issue fast as Issue  template already created for this repo .
Thank  u

**Comment by GundaSudarrshan** [comment]
@Vikash-8090-Yadav  Updated pls check it now

**Comment by Vikash-8090-Yadav** [comment]
Great :smile:

**Pull request merged**